circus-train-avro/src/main/java/com/hotels/bdp/circustrain/avro/transformation/AbstractAvroSerDeTransformation.java

```diff
@@ -85,8 +91,15 @@ public void tableReplicationStart(EventTableReplication tableReplication, String
   }
 
   @Override
-  public void tableReplicationSuccess(EventTableReplication tableReplication, String eventId) {}
+  public void tableReplicationSuccess(EventTableReplication tableReplication, String eventId) {
+    this.eventId = null;
```

Comment:
Why was this changed ?

Reply:
Just resetting the values after a completed replication. Once the replication is done these value are not valid anymore another replication might come in. This state should be refreshed for every replication.

Reply:
I stand correct I shouldn't have changed this all the tests started failing. I've put this back I guess it's ok that we override on start.